IPython/html/static/widgets/js/widget_bool.js

```diff
@@ -129,10 +129,11 @@ define([
 
                 var description = this.model.get('description');
                 this.$el.attr("title", this.model.get("tooltip"));
-                if (description.trim().length === 0) {
+                this.$el.text(description);
+                var icon = this.model.get("icon");
```

Comment:
I see from the Python side that icon can't reasonably be anything other than a string, so this assumption should be okay.